src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -907,6 +907,79 @@ public override IOperation VisitWhileLoop(IWhileLoopOperation operation, int? ca
             return null;
         }
 
+        public override IOperation VisitVariableDeclarationGroup(IVariableDeclarationGroupOperation operation, int? captureIdForResult)
+        {
+            // Anything that has a declaration group (such as for loops) needs to handle them directly itself, this should only be encountered by the visitor for declaration statements.
```

Comment:
Consider splitting the comment across multiple lines. Same for other long lines in this method.

Reply:
The "advantages" of the new monitors.

---
In reply to: [167056137](https://github.com/dotnet/roslyn/pull/24692#discussion_r167056137) [](ancestors = 167056137)